src/scripts/dungeons/Dungeon.ts

```diff
@@ -145,95 +145,95 @@ dungeonList['Sprout Tower'] = new Dungeon('Sprout Tower',
     ['Rattata', 'Gastly', 'Hoothoot'],
     [GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.Item_magnet],
     28735,
-    [new DungeonBossPokemon('Bellsprout', 2000, 10)],
+    [new DungeonBossPokemon('Bellsprout', 120000, 10)],
     2500, BadgeCase.Badge.Elite_KantoChampion, 31, 5
 );
 
 dungeonList['Ruins of Alph'] = new Dungeon('Ruins of Alph',
     ['Natu', 'Wooper', 'Quagsire', 'Smeargle', 'Magikarp', 'Poliwag', 'Poliwhirl'],
     [GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.Item_magnet],
-    600,
-    [new DungeonBossPokemon('Unown', 3000, 14)],
+    30600,
+    [new DungeonBossPokemon('Unown', 130000, 14)],
     3000, BadgeCase.Badge.Zephyr, 32, 7
 );
 
 dungeonList['Union Cave'] = new Dungeon('Union Cave',
     ['Rattata', 'Sandshrew', 'Zubat', 'Geodude', 'Onix', 'Goldeen', 'Magikarp'],
     [GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.Item_magnet],
-    600,
-    [new DungeonBossPokemon('Wooper', 3000, 14)],
+    31600,
+    [new DungeonBossPokemon('Wooper', 130000, 14)],
     3000, BadgeCase.Badge.Zephyr, 32, 7
 );
 
 dungeonList['Slowpoke Well'] = new Dungeon('Slowpoke Well',
     ['Zubat', 'Slowpoke'],
     [GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.xExp],
-    900,
-    [new DungeonBossPokemon('Slowbro', 4000, 20)],
+    33900,
+    [new DungeonBossPokemon('Slowbro', 140000, 20)],
     3500, BadgeCase.Badge.Zephyr, 33, 12
 );
 
 dungeonList['Ilex Forest'] = new Dungeon('Ilex Forest',
     ['Caterpie', 'Metapod', 'Weedle', 'Kakuna', 'Zubat', 'Oddish', 'Paras', 'Hoothoot'],
     [GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.Token_collector],
-    1200,
-    [new DungeonBossPokemon('Noctowl', 5000, 30), new DungeonBossPokemon('Beedrill', 5000, 30), new DungeonBossPokemon('Butterfree', 5000, 30), new DungeonBossPokemon('Celebi', 300000, 50)],
+    41200,
+    [new DungeonBossPokemon('Noctowl', 150000, 30), new DungeonBossPokemon('Beedrill', 5000, 30), new DungeonBossPokemon('Butterfree', 5000, 30), new DungeonBossPokemon('Celebi', 300000, 50)],
```

Comment:
The other boss Pokemon in this dungeon also need to be modified.

_Same thing for the other dungeons that have been changed

Reply:
Whoops

Reply:
Fixed